### PR TITLE
convert @babel/helper-fixtures to typescript

### PR DESCRIPTION
**Pull request by zxbodya**
| Q                        | A 
| ------------------------ | ---
| License                  | MIT

@babel/helper-fixtures part of #11578

### CODE REVIEW

File: packages/babel-helper-fixtures/src/index.ts [hunk 1]
```diff
@@ -119,7 +122,7 @@ function pushTask(taskName, taskDir, suite, suiteName) {
   const taskOptsLoc = tryResolve(taskDir + "/options");
   if (taskOptsLoc) Object.assign(taskOpts, require(taskOptsLoc));
 
-  const test = {
+  const test: any = {
```
Comment (nicolo-ribaudo) [inline packages/babel-helper-fixtures/src/index.ts:125]: Doesn't TS infer the correct type here?

Comment (zxbodya) [reply packages/babel-helper-fixtures/src/index.ts:125]: it does, but there are additional properties assigned later(`sourceMappings`, `sourceMap`, `inputSourceMap`)

other options would be - restructure the code to construct `test` with all properties in it, or to have type for it separately

Comment (zxbodya) [reply packages/babel-helper-fixtures/src/index.ts]: made a bit better fix instead of any

**Review by JLHwung — APPROVED** [review]

**Review by nicolo-ribaudo — APPROVED** [review]
Thanks!

**Comment by nicolo-ribaudo** [comment]
I'm merging this as a single commit (not rename + convert) because git is correctly tracking renamed files.

**Pull request merged**